e2e/tests/transfer/incentivized2_test.go

```diff
@@ -0,0 +1,266 @@
+//go:build !test_e2e
+
+package transfer
+
+import (
+	"context"
+	"testing"
+
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	test "github.com/strangelove-ventures/interchaintest/v8/testutil"
+	testifysuite "github.com/stretchr/testify/suite"
+
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+	feetypes "github.com/cosmos/ibc-go/v8/modules/apps/29-fee/types"
+	transfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+func TestIncentivized2TransferTestSuite(t *testing.T) {
```

Comment:
what's the reason we need a `TestIncentivized2TransferTestSuite`? Is it okay to stick with a single test suite for incenvitizes transfers? 

If it is a requirement to have a new test suite, I think we need to decide on a better name, as this doesn't really tell us anything.

Reply:
the problem here is that they will interact with each other if we put them together in a test suite, same reason why we are missing t.Parallel(). Do you have any ideas?

Reply:
Ah, I think it is fine if they are in a different test suite, but I think we can maybe give them better names.